graphqlapi/src/main/java/io/stargate/graphql/schema/NameConversions.java

```diff
@@ -0,0 +1,240 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.graphql.schema;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+
+public class NameConversions {
+
+  enum IdentifierType {
+    TABLE,
+    UDT,
+    COLUMN,
+  }
+
+  private NameConversions() {}
+
+  public static String toGraphql(String cqlName, IdentifierType type) {
+    if (cqlName == null || cqlName.isEmpty()) {
+      throw new IllegalArgumentException("CQL name must be non-null and not empty");
+    }
+    if (type == IdentifierType.TABLE) {
+      for (Map.Entry<String, String> entry : RESERVED_GRAPHQL_NAMES.entrySet()) {
```

Comment:
Any reason not to use `RESERVED_GRAPHQL_NAMES.get(cqlName)` here?

Reply:
No 😳 
I'll change it.